openedx/features/colaraz_features/api/serializers.py

```diff
@@ -0,0 +1,61 @@
+"""
+Serializers for colaraz API.
+"""
+from django.contrib.sites.models import Site
+
+from rest_framework import serializers
+from organizations.models import Organization
+
+from openedx.features.colaraz_features.api.fields import DomainField
+from openedx.features.colaraz_features.helpers import (
+    create_sites, create_site_themes, create_organization, create_site_configurations,
+)
+
+
+class SiteOrgSerializer(serializers.Serializer):
+    site_domain = DomainField()
+    site_name = serializers.CharField(max_length=255)
+    site_theme = serializers.CharField(max_length=255)
+    org_name = serializers.CharField(max_length=255)
+    org_short_name = serializers.SlugField(max_length=255)
+    platform_name = serializers.CharField(max_length=255)
+    university_name = serializers.CharField(max_length=255, required=False)
+    organizations = serializers.ListField(
+        child=serializers.SlugField(max_length=255)
+    )
+
+    @staticmethod
+    def validate_site_domain(value):
+        """
+        Validate that site domain does not already exists.
+        """
+        if Site.objects.filter(domain=value).exists():
+            raise serializers.ValidationError('A site with domain "{}" already exists.'.format(value))
+        return value
+
+    @staticmethod
+    def validate_org_name(value):
```

Comment:
Don't you think we need to check the organization short name too?

Reply:
Done